### ISSUE TITLE
Create a icon to represent Chocolatey templates

### ISSUE THREAD

**Issue by gep13** [chocolatey-community/chocolatey-templates#11]
To make it clear on community.chocolatey.org that these packages are part of the building blocks for using Chocolatey to create packages.

@TheCakeIsNaOH @AdmiringWorm I am not an artist/designer, so this is not something that I would be able to come up with. But perhaps @designxsteph might be able to help here?

**Comment by st3phhays** [comment]
@gep13 Is this the icon that will show up on the packages page, or where will this icon be used? Would something similar to the new SHIM icon work here?

**Comment by gep13** [comment]
@st3phhays yes, this would be the icon that would appear in the package page on the Chocolatey Community Repository.

Yes, something similar to what we did with the shim icon might work well here.  One other suggestion might be something similar to what we did with the Cake-Contrib logo:

https://github.com/cake-contrib/graphics

Here we were trying to play with the idea that the Cake-Contrib projects were about putting the building blocks, or jigsaw pieces, in place to build on top of Cake.  The concept of templates/extensions, is a similar idea in my head.
